L1Trigger/L1THGCal/python/hgcalTriggerPrimitiveDigiProducer_cfi.py

```diff
@@ -1,28 +1,34 @@
 import FWCore.ParameterSet.Config as cms
 
 import SimCalorimetry.HGCalSimProducers.hgcalDigitizer_cfi as digiparam
+import RecoLocalCalo.HGCalRecProducers.HGCalUncalibRecHit_cfi as recoparam
+import RecoLocalCalo.HGCalRecProducers.HGCalRecHit_cfi as recocalibparam 
 
 fe_codec = cms.PSet( CodecName  = cms.string('HGCalTriggerCellBestChoiceCodec'),
                      CodecIndex = cms.uint32(2),
                      NData = cms.uint32(12),
                      MaxCellsInModule = cms.uint32(116),
-                     DataLength = cms.uint32(8),
+                     DataLength = cms.uint32(16),
```

Comment:
Keep the same default parameter as before: 8
It should be modified in test configs if needed